.github/workflows/push.yaml

```diff
@@ -27,14 +29,26 @@ jobs:
 
       - name: Run shellcheck on all scripts
         run: make shellcheck
```

Comment:
These should be removed right? As they have moved into `lint-shell`?
```suggestion
```

Reply:
No longer. I will take care of that split in some other PR :)